datafusion/sqllogictest/src/engines/conversion.rs

```diff
@@ -149,19 +128,41 @@ mod tests {
 
     #[test]
     fn test_big_decimal_to_str() {
```

Comment:
Do we still need these tests if `to_plain_string` is reliable🤔

Reply:
maybe not, but this also tests for normalization if any, and this may be type dependent
this test and the tested helper were supposed to go away, see last commit in https://github.com/apache/datafusion/pull/13743. Hopefully this lands one day.